src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/controller/PersonControllerIntTest.kt

```diff
@@ -174,7 +174,6 @@ class PersonControllerIntTest() : IntegrationTestBase() {
       familyName = "Jones",
       dateOfBirth = LocalDate.of(1968, 8, 15),
       otherIdentifiers = OtherIdentifiers(
-        pncIdentifier = PNCIdentifier("1976/1234Z"),
```

Comment:
This test was a new one on me. I don't think we are anywhere near supporting this functionality and based on the architecture diagram, it may not even be in this application. I think it might be an idea to remove it for the moment but have bodged it for now. If you supply a PNC it won't work.